### PR TITLE
[FIX] mass_mailing: remove extra domain in view

### PR DESCRIPTION
**Pull request by jeh-odoo**
Remove the extra mailing_domain in the form view.

task-3562293

### CODE REVIEW

File: addons/mass_mailing/views/mailing_mailing_views.xml [hunk 1]
```diff
@@ -235,18 +235,15 @@
...
-                                    <div name="mass_mailing_domain" invisible="mailing_model_name == 'mailing.list'"
-                                        class="col-md-6 d-block">
-                                        <field name="mailing_domain" widget="domain" options="{'model': 'mailing_model_real'}" readonly="state in ('sending', 'done')"/>
-                                    </div>
                                 </div>
 
                                 <field name="mailing_model_name" invisible="1"/>
                                 <field name="mailing_on_mailing_list" invisible="1"/>
                                 <field name="mailing_model_real" invisible="1"/>
-                                <div invisible="mailing_on_mailing_list">
```
Comment (awa-odoo) [inline addons/mass_mailing/views/mailing_mailing_views.xml:247]: Strange, could we have more information on why this was duplicated? And a link to the "incriminated commit" in the commit message?

File: addons/mass_mailing/views/mailing_mailing_views.xml [hunk 2]
```diff
@@ -235,18 +235,15 @@
...
-                                    <field name="mailing_domain" widget="domain" options="{'model': 'mailing_model_real', 'foldable': true}"
-                                    readonly="state in ('sending', 'done')"/>
+                                <div name="mass_mailing_domain" invisible="mailing_on_mailing_list">
```
Comment (awa-odoo) [inline addons/mass_mailing/views/mailing_mailing_views.xml:243]: Do we need the extra div then? Can't we put the invisible attribute on the field directly?

**Review by awa-odoo — COMMENTED** [review]
Hello @jeh-odoo 

A couple comments.
From what I understand, if we squeeze this one before the freeze, then we don't have to target stable, so would be nice (even though not crucial).

Thanks & cheers!

File: addons/mass_mailing/views/mailing_mailing_views.xml (hunk 1 above)
Comment (jeh-odoo) [reply addons/mass_mailing/views/mailing_mailing_views.xml:257]: Done :+1:

File: addons/mass_mailing/views/mailing_mailing_views.xml (hunk 2 above)
Comment (jeh-odoo) [reply addons/mass_mailing/views/mailing_mailing_views.xml]: Indeed we can remove it :+1:

**Review by jeh-odoo — COMMENTED** [review]
Hello @awa-odoo 
Change done and I added the commit "responsible" in the commit message with a short description.
Cheers!

**Comment by awa-odoo** [comment]
@robodoo r+

**Pull request closed without merging**